tests/acceptance/animatable-test.js

```diff
@@ -21,7 +21,7 @@ module('Acceptance: modal-dialog | animatable', function (hooks) {
   test('basic modal', async function (assert) {
     assert.dom(modalRootElementSelector).exists({ count: 1 });
     assert.isAbsent(overlaySelector);
-    assert.dom('#example-basic button').exists({ count: 1 });
+    assert.dom('#example-basic button').exists({ count: 2 });
```

Comment:
The number of `button` changed to 2 because `ember-shiki` has a copy button. To change the number back to 1, consider using `data-test-*` selectors instead of IDs and class names.